src/app/api/run/[runId]/route.ts

```diff
@@ -0,0 +1,21 @@
+import { NextResponse } from 'next/server'
+import { loadResearchForReview } from '@/app/utils'
+
+export async function GET(request: Request, { params }: { params: { runId: string } }) {
```

Comment:
What's the user for this API? I'm worried about the API surface area that we have to maintain and document.